.gitignore

```diff
@@ -4,3 +4,4 @@ coverage/
 npm-debug.log
 yarn-debug.log
 yarn-error.log
+cli.js
```

Comment:
not sure if intentional?

Reply:
Yeah it was. I hope to find to include the built version only upon release, not as a part of the repo itself

Reply:
one thing that would suck is for every PR, we would have to submit a built version as well

Reply:
Yeah. Maybe it's more ideal to keep it JS, and use jsdocs as the typechecker with TS.

https://github.com/Microsoft/TypeScript/wiki/JsDoc-support-in-JavaScript

Reply:
I think adding a build system just increases the complexity and means more maintenance on our ends so keeping it vanilla may be easier for us to manage in the long term